### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by MaksymKorniiov**

### CODE REVIEW

**Comment by OrenMe** [comment]
@MaksymKorniiov please avoid naming PR with version name, nothing guarantees this will be a part of any version so this can be misleading.
When naming a PR please make it as informative and short as possible in the title, and add description in the body(first comment) describing the change

File: modules/KalturaSupport/components/volumeControl.js [hunk 1]
```diff
@@ -158,6 +158,7 @@ mw.PluginManager.add( 'volumeControl', mw.KBaseComponent.extend({
 			var title = gM('mwe-embedplayer-volume-value', percent );
             var $slider = this.getSlider().find('a');
             this.setAccessibility($slider, title);
+            this.updateVolumeUI(percent/100);
```
Comment (eitanavgil) [inline modules/KalturaSupport/components/volumeControl.js]: @MaksymKorniiov  
Did we check if the volume comes from the user cookie it is being updated accordingly?

File: modules/Quiz/resources/css/quiz.css [hunk 2]
```diff
@@ -231,7 +233,7 @@
     cursor: default!important;
 }
 .display-question:focus{
-    outline: none;
+    /*outline: none;*/
```
Comment (eitanavgil) [inline modules/Quiz/resources/css/quiz.css]: @MaksymKorniiov  
If we decide that this is not relevant anymore - let's delete it.

File: modules/KalturaSupport/components/volumeControl.js (hunk 1 above)
Comment (MaksymKorniiov) [reply modules/KalturaSupport/components/volumeControl.js]: @eitanavgil Yes

File: modules/Quiz/resources/css/quiz.css (hunk 2 above)
Comment (MaksymKorniiov) [reply modules/Quiz/resources/css/quiz.css]: @eitanavgil fixed it in new commit

File: modules/KalturaSupport/components/logo.js [hunk 3]
```diff
@@ -9,9 +9,9 @@
 			displayImportance: 'low',
 			align: "right",
 			cssClass: "kaltura-logo",
-			href: null,
-			title: null,
-			img: null
+			href: 'http://www.kaltura.com',
+			title: 'Kaltura',
+			img: 'http://kgit.html5video.org/branches/master//skins/kdark/images/kalturaLogo.png'
```
Comment (eitanavgil) [inline modules/KalturaSupport/components/logo.js]: @MaksymKorniiov  
We cannot hard-code a release with path to kgit. Let's try to change this to a relative path and if not possible I will work with you on an absolute kaltura CDN file.

**Pull request closed without merging**